kobo/apps/trackers/submission_utils.py

```diff
@@ -0,0 +1,139 @@
+import os
```

Comment:
This file merits broader review. Let's call it out of scope of this PR and we can change it later too. I'd love to discuss consistent generation of test data.